lib/thrift/binary/framed/client.ex

```diff
@@ -155,45 +157,51 @@ defmodule Thrift.Binary.Framed.Client do
     {:connect, :reconnect, %{s | sock: nil}}
   end
 
-  @spec oneway(pid, data) :: :ok
+  @spec oneway(pid, String.t, data, options) :: :ok
   @doc """
   Execute a one way RPC. One way RPC calls do not generate a response,
   and as such, this implementation uses `GenServer.cast`.
   The data argument must be a properly formatted Thrift message.
   """
-  def oneway(conn, data) do
-    Connection.cast(conn, {:oneway, data})
+  def oneway(conn, rpc_name, serialized_args, _opts) do
+    :ok = Connection.cast(conn, {:oneway, rpc_name, serialized_args})
+    :ok
   end
 
-  @spec request(pid, data, options) :: protocol_response
+  @spec call(pid, String.t, data, options) :: protocol_response
   @doc """
   Executes a Thrift RPC. The data argument must be a correctly formatted
   Thrift message.
 
   The `options` argument takes the same type of keyword list that `start_link` takes.
   """
-  def request(conn, data, options) do
-    tcp_opts = Keyword.get(options, :tcp_opts, [])
-    gen_server_opts = Keyword.get(options, :gen_server_opts, [])
+  def call(conn, rpc_name, serialized_args, opts) do
```

Comment:
Yeah that's a good point. `request` was confusing because I was looking around for `call` and `oneway` functions. How about `thrift_call` and `thrift_oneway`?